src/lib/dnssd/Resolver.h

```diff
@@ -192,44 +202,48 @@ struct DiscoveredNodeData
         {
             ChipLogDetail(Discovery, "\tPairing Hint: %u", pairingHint);
         }
-        if (!IsHost(""))
-        {
-            ChipLogDetail(Discovery, "\tHostname: %s", hostName);
-        }
         if (!IsInstanceName(""))
         {
             ChipLogDetail(Discovery, "\tInstance Name: %s", instanceName);
         }
-        for (unsigned j = 0; j < numIPs; j++)
-        {
-#if CHIP_DETAIL_LOGGING
-            char buf[Inet::IPAddress::kMaxStringLength];
-            char * ipAddressOut = ipAddress[j].ToString(buf);
-            ChipLogDetail(Discovery, "\tIP Address #%d: %s", j + 1, ipAddressOut);
-            (void) ipAddressOut;
-#endif // CHIP_DETAIL_LOGGING
-        }
-        if (port > 0)
-        {
-            ChipLogDetail(Discovery, "\tPort: %u", port);
-        }
         ChipLogDetail(Discovery, "\tCommissioning Mode: %u", commissioningMode);
-        if (mrpRetryIntervalIdle.HasValue())
-        {
-            ChipLogDetail(Discovery, "\tMrp Interval idle: %" PRIu32 " ms", mrpRetryIntervalIdle.Value().count());
-        }
-        else
-        {
-            ChipLogDetail(Discovery, "\tMrp Interval idle: not present");
-        }
-        if (mrpRetryIntervalActive.HasValue())
-        {
-            ChipLogDetail(Discovery, "\tMrp Interval active: %" PRIu32 " ms", mrpRetryIntervalActive.Value().count());
-        }
-        else
-        {
-            ChipLogDetail(Discovery, "\tMrp Interval active: not present");
-        }
+    }
+};
+
+struct ResolvedNodeData
+{
+    CommonResolutionData resolutionData;
+    OperationalNodeData operationalData;
+
+    void LogNodeIdResolved() const
+    {
+#if CHIP_PROGRESS_LOGGING
+        // Would be nice to log the interface id, but sorting out how to do so
+        // across our differnet InterfaceId implementations is a pain.
```

Comment:
```suggestion
        // across our different InterfaceId implementations is a pain.
```
(I know, this just got moved...)